### PR TITLE
Claim main differences with linux container runtimes

### PR DESCRIPTION
**Pull request by resouer**
We need to claim main differences with linux container runtimes, so developers will not get confused.

### CODE REVIEW

**Comment by feiskyer** [comment]
Fixing cni problem via #157

File: README.md [hunk 1]
```diff
@@ -113,6 +113,27 @@ Further information could be found at:
...
+  - This behavior is configurable by `--defaultCPUNum` and `--defaultMemoryMB`  of frakti
+- Mixed Runtimes Mode
+  - frakti support mixed runtimes on the same Node (HyperContainer and Docker). We recommend user to run `BestEffort` Pods, daemon Pods in Docker runtime by adding `runtime.frakti.alpha.kubernetes.io/OSContainer` annotation to them
+  - Additionally, special cases like privileged Pods, host network Pods etc will be automatically run in Docker runtime
+- Persistent Volume
+  - All k8s PVs are supported in frakti
+  - We are also working on another build-in PV plugin for frakti and it will attach block devices to Pod as volumes directly to achieve much higher performance.
+- Cross-host Networking
+  - frakti is fully based on CNI (bridge mode only for now), so there's no big difference here
+  - We are working on making `Fannel` & `Calico` work out-of-box with `fakti` based Kubernetes
```
Comment (feiskyer) [inline README.md]: s/fakti/frakti

Comment (resouer) [inline README.md]: fixed

**Pull request merged**